test_unstructured/metrics/test_table_formats.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from unstructured.metrics.table.table_formats import SimpleTableCell
+
+
+def test_simple_table_cell_parsing_from_table_transformer_when_expected_input():
+    table_transformer_cell = {"row_nums": [3, 2, 1], "column_nums": [6, 7], "cell text": "text"}
```

Comment:
lets use parametrize and add two more test cases:
- row nums has only 1 value
- column nums has only 1 value